### DISCUSSION TITLE
tailwind 3 doesn't apply classes for components

### DISCUSSION THREAD

**Discussion by Mnigos** [tailwindlabs/tailwindcss#6531] [Help]
**What version of Tailwind CSS are you using?**

v3.0.0

**What build tool (or framework if it abstracts the build tool) are you using?**

Remix.js 1.0.6

**What version of Node.js are you using?**

v16.10.0

**What browser are you using?**

Chrome

**What operating system are you using?**

Windows

**Reproduction URL**

[reproduction](https://github.com/Mnigos/example-for-tailwind-issue)

**Describe your issue**

Tailwind classes like `text-red-500` don't apply on components' on version `>=3.0.0`. If I use `2.2.19` version classes applies correctly.

**Comment by nbgoodall** [comment]
The tailwind styles aren't being generated; have you followed the Remix docs — https://remix.run/docs/en/v1/guides/styling#tailwind?

**Comment by adamwathan** [comment]
Looks like Tailwind isn’t actually configured to run because it’s not included in your package.json scripts. Take a look at the Remix docs as mentioned above.

**Reply by Mnigos** [reply]
Yes sorry I forgot to push changes with scripts. Now it's on repo.

**Reply by nbgoodall** [reply]
The config isn't loading correctly — if you change the `tailwind.config.ts` to `tailwind.config.js` (TS -> JS) the server will pick it up automatically, but it'll error so you'll need to change `export default` to `module.exports = `. Also, Change `purge` to `content`; [a few options changed in v3](https://tailwindcss.com/docs/configuration#creating-your-configuration-file). :+1:

**Reply by Mnigos** [reply]
yes it works now. Thanks
